packages/js/src/utils/requester.ts

```diff
@@ -1,32 +1,20 @@
 import { isNode } from '@silverhand/essentials';
 
-import { LogtoError } from './errors';
+import { LogtoError, LogtoRequestError, LogtoRequestErrorParameters } from './errors';
 
-interface LogtoErrorResponse {
-  error?: string;
-  error_description?: string;
-}
-
-interface LogtoErrorParameters extends LogtoErrorResponse {
-  message?: string;
-}
-
-const getLogtoErrorParametersByResponse = async (
-  response: Response
-): Promise<LogtoErrorParameters> => {
+const getLogtoResponseError = async (response: Response): Promise<LogtoRequestError> => {
   const text = await response.text();
   try {
-    const data = JSON.parse(text) as LogtoErrorResponse;
-    if (data.error || data.error_description) {
-      return {
-        error: data.error,
-        error_description: data.error_description,
-      };
-    }
-
-    return { message: response.statusText };
+    const { code, message } = JSON.parse(text) as LogtoRequestErrorParameters;
```

Comment:
1. use response.json to get the json format return data. should not JSON.parse(text) here.
2. Let's use ts declaration file to update the body interface definition of json:
```
interface Body {
  json<T>(): Promise<T>;
}
```
Instead of as

Reply:
Added, PTAL.